### DISCUSSION TITLE
Extract MS1 feature count from output

### DISCUSSION THREAD

**Discussion by ahmohamed** [vdemichev/DiaNN#239] [Q&A] [answered]
Thanks Vadim for such a superb tool.

@jkelabora and I (WEHI Proteomics / Andrew Webb's lab) were wondering if it's possible to extract the number of MS1 features from the output. We usuaully use these features to evaluate / optimize different LC methods. I can see that the main report and counts in report_stats.tsv include only identified precusors. There is also ms1_intensity column, but I'm not quite sure whether it's calculated based on identified features only as well. Can you please clarify?

I also have a few other questions, for which I'll create different topics.

Thanks again for your help.
Cheers.
Ahmed.

**Comment by vdemichev** [comment, marked as answer]
Hi Ahmed,

What do you mean by 'ms1 features'?
MS1 intensity in the stats file is the total signal (sum of all peak intensities) recorded in the MS1 mode within the run.

Best,
Vadim

**Reply by ahmohamed** [reply]
By MS1 features I mean all MS1 precursors (both successfully matched and non-matched). In DDA mode, we usually use MaxQuant allpeptides.txt for that. Not sure what's the equivalent (if any) in DIA-NN output. I'll have another look at ms1_intensity column.

**Reply by ahmohamed** [reply]
Thanks again Vadim. I've looked at the MS1.Signal and it seems to give us what we need (indication of the MS1 features even when samples have low IDs, for example in blanks).
